crates/sdk/src/rpc.rs

```diff
@@ -1518,3 +1519,25 @@ pub async fn query_ibc_denom<N: Namada>(
 
     token.as_ref().to_string()
 }
+
+/// Query the IBC rate limit for the provided token
+pub async fn query_ibc_rate_limit<C: Client + Sync>(
```

Comment:
An alternative to this function could be to add another path to the `TOKEN` https://github.com/anoma/namada/blob/9eb747ce145d622fda5296475c1d35eb4c3e86a4/crates/sdk/src/queries/vp/token.rs#L15

Reply:
don't think we can add a router path now that mainnet has been launched, perhaps @tzemanovic can comment here

Reply:
You can add more paths, but then the query only works against a node that is updated to contain that (additions are not breaking changes). Still, if it can be implemented without extra path then that's preferable. You can also implement it as a client-only query if you prefer to have it as a method on router though it's not much different from this (see `mod client_only_methods` in e.g. `crates/sdk/src/queries/vp/token.rs` )